src/components/RoleButton/RoleButton.js

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types';
 import Button from '../Button';
 import ArrowIcon from './ArrowIcon';
 
-function ButtonBase({ children, ...props }) {
+function RoleButton({ children, ...props }) {
```

Comment:
I would consider renaming it to something more generic since it is no longer associated with Role only, for example **RedirectButton** (and add comment on top of component: A button with fat arrow :)